cmd/zpool/zpool_main.c

```diff
@@ -1766,7 +1766,7 @@ print_import_config(const char *name, nvlist_t *nv, int namewidth, int depth,
  */
 static void
 print_logs(status_cbdata_t *cb, zpool_handle_t *zhp, nvlist_t *nv,
-    int namewidth, boolean_t verbose, int name_flags)
+    int namewidth, int name_flags)
```

Comment:
@inkdot7 just saw your response after I posted.

I'd prefer a "`generic_cbdata_t`" or "`format_cbdata_t`" for storing the formatting parameters, rather than resorting to casting tricks.  Seems like a lot of different zpool command use the same generic verbose/name_flags/namewidth variables.  You could embed it in the parent struct like:

```
struct status_cbdata_t {
	struct generic_cbdata_t	cb;
	...
} scb;
```
Then just pass around `cb` to all the functions.  If you had a function that actually needed the parent struct, you could use `container_of()` on `cb` to get to it (you'd have to add the userspace  `container_of()` though).   Or alternatively just pass around the parent struct instead of `cb`.